how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md

```diff
@@ -56,4 +56,6 @@ then you are using the new role system and can go ahead with the upgrade to v8.
 In your code, if you were using the ```ForestLiana::PermissionsChecker``` for check the permission on overriding route.
 You need to replace ```PermissionsChecker.new(...)``` by ```ForestLiana::Ability::forest_authorize!(action, forest_user, @resource)```.
 
+You must also ensure that the controllers that manage your smart actions extends from the `ForestLiana::SmartActionsController` controller.
```

Comment:
```suggestion
You must also ensure that the controllers that manage your smart actions extend from the `ForestLiana::SmartActionsController` controller.
```

That sentence should also be placed under the next one I think, because the link below related to route override, not smart action.

Reply:
good insight !